packages/shared-base/src/SingletonModal/index.ts

```diff
@@ -116,6 +116,16 @@ export class SingletonModal<
             this.onAbort = (error) => reject(error)
         })
     }
+
+    __unsafe_overwrite_methods__(methods: {
```

Comment:
can you document the purpose of this function?

Reply:
hello? cc @UncleBill

Reply:
It might be written by @guanbinrui